build/testdata/055.in

```diff
@@ -1,4 +1,4 @@
-load(":foo.bzl","foo",a="a",b="c","d")
+load(":foo.bzl","foo",a="a",b="c","d", c="b")
```

Comment:
is this even a valid parse?  non kw args intermixed with kw?

Reply:
It's valid for load statements, however breaks the compatibility with python tools, also IMO not readable well.